src/libsyntax/feature_gate/builtin_attrs.rs

```diff
@@ -502,9 +502,14 @@ pub const BUILTIN_ATTRIBUTES: &[BuiltinAttribute] = &[
         rustc_test_marker, Normal, template!(Word),
         "the `#[rustc_test_marker]` attribute is used internally to track tests",
     ),
+    gated!(
```

Comment:
Looks mostly good but I think we can use `rustc_attr!(...)` here cause the attribute is meant to be perma-unstable and so we don't need a special feature gate for it (also results in marginally less cost for the compiler).

Reply:
@Centril I have no experience with this. Would you mind making the appropriate change? (You can just push to my branch.)

Reply:
Guess you need to rename the attribute too.

Reply:
You can copy the invocation above for `rustc_test_marker`:
```rust
rustc_attr!(
    // Used by interpreters:
    rustc_interp_user_fn, Normal, template!(Word),
    "`#[interp_user_fn]` is for use by interpreters only"
),
```
and it should just work -- you'll need to `--bless` the feature gate test also.

You are correct that `rustc_` should be added also as a prefix. :)